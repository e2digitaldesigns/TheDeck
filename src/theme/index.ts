export * from "./styles/forms.styles";
export * from "./styles/scrollBars.styles";
