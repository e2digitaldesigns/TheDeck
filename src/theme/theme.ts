export const theme = {
  sizes: {
    application: {
      height: "785px",
      width: "1280px"
    },
    header: {
      height: "2rem",
      width: "100vw"
    },
    macroDeck: {
      sideBar: { width: "240px" },
      mainContent: { width: "1016px" }
    }
  },
  colors: {
    body: "black",
    border: "#111",
    accent1: "#f542e9",
    accent2: "#291697",
    accent3: "#404040",
    accent4: "#2b2b2b",
    accent5: "#1a1a1a",
    gridBg: "43, 45, 48,",
    font: "white",
    main: "#100727",
    mainBackground: "black",
    scrollBackground: "#100727",
    scrollFont: "#dedede",
    scrollHeader: "black",
    scrollHeaderFont: "white"
  }
};
