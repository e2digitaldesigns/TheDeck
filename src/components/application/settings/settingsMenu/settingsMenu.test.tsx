import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import { HashRouter as Router } from "react-router-dom";
import SettingsMenu from "./settingsMenu";
import { ThemeProvider } from "styled-components";
import { theme } from "../../../../theme/theme";

const setup = () => {
  return render(
    <ThemeProvider theme={theme}>
      <Router>
        <SettingsMenu />
      </Router>
    </ThemeProvider>
  );
};

describe("<SettingsMenu />", () => {
  it("Should render correctly", () => {
    const wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
