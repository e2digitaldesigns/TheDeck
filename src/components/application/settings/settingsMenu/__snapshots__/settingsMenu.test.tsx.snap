// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SettingsMenu /> Should render correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <h1
        class="sc-gKclnd fOPFgh"
      >
        Settings
      </h1>
      <ul
        class="sc-iCfMLu bZjVHc"
      >
        <li
          class="sc-furwcr skDFI"
        >
          <a
            href="#/settings/macroDeck"
          >
            MacroDeck
          </a>
        </li>
        <li
          class="sc-furwcr skDFI"
        >
          <a
            href="#/settings/server"
          >
            Server
          </a>
        </li>
        <li
          class="sc-furwcr skDFI"
        >
          <a
            href="#/settings/obs"
          >
            OBS
          </a>
        </li>
        <li
          class="sc-furwcr skDFI"
        >
          <a
            href="#/settings/twitch"
          >
            Twitch
          </a>
        </li>
      </ul>
    </div>
  </body>,
  "container": <div>
    <h1
      class="sc-gKclnd fOPFgh"
    >
      Settings
    </h1>
    <ul
      class="sc-iCfMLu bZjVHc"
    >
      <li
        class="sc-furwcr skDFI"
      >
        <a
          href="#/settings/macroDeck"
        >
          MacroDeck
        </a>
      </li>
      <li
        class="sc-furwcr skDFI"
      >
        <a
          href="#/settings/server"
        >
          Server
        </a>
      </li>
      <li
        class="sc-furwcr skDFI"
      >
        <a
          href="#/settings/obs"
        >
          OBS
        </a>
      </li>
      <li
        class="sc-furwcr skDFI"
      >
        <a
          href="#/settings/twitch"
        >
          Twitch
        </a>
      </li>
    </ul>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
